TinyInsights/ApplicationInsightsProvider.cs

```diff
@@ -46,6 +47,18 @@ public ApplicationInsightsProvider(string connectionString)
         }
 
         Task.Run(SendCrashes);
+
+        if(IsAutoTrackPageViewsEnabled)
```

Comment:
This will always be true here because the property is set after the constructor has finished.